docs/README.md

```diff
@@ -13,6 +13,9 @@ access it in the browser.
 - Use cloud servers to speed up tests, compilations, downloads, and more
 - Preserve battery life when you're on the go; all intensive tasks run on your
   server
+  
+| 🔔 code-server is the free and libre single-user variant of [Coder](https://coder.com/), our enterprise developer workspace platform. For more information, visit [Coder.com](https://coder.com/docs/comparison)
```

Comment:
What is the `|` for? Do we need that?

Reply:
Yeah, idk but I think it helps making the box.

Reply:
Wait, I understood! In Markdown, you need the `|` to make a table, right? We made a 1x1 table aka. the banner that has the alert! In Markdown, you make tables like this:
```markdown
|      |      |      |
| ---- | ---- | ---- |
|      |      |      |
|      |      |      |
|      |      |      |
```

How it looks:
|      |      |      |
| ---- | ---- | ---- |
|      |      |      |
|      |      |      |
|      |      |      |


So, we used this:
```markdown
| Hello
| ---
```
And it looks like this:
| Hello
| ---

Do you get it?

Reply:
OH! That makes sense. Let's keep it then 👍  Thanks for the explanation and demo :)